dts/riscv/ite/it81xx2.dtsi

```diff
@@ -0,0 +1,506 @@
+/*
+ * Copyright (c) 2023 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Can you add `#include "it8xxx2.dtsi` here?  This avoids every board having to include both the chip-specific DTS and the common DTS file.

Reply:
Agree, and the boards in CROS need to be adjusted as:
https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3936234